src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Metadata/Models/ContentTypeExtensions.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using OrchardCore.ContentManagement.Metadata.Settings;
+
+namespace OrchardCore.ContentManagement.Metadata.Models;
+
+public static class ContentTypeExtensions
+{
+    public static bool HasStereotype(this ContentTypeDefinition contentTypeDefinition)
+    {
+        return !String.IsNullOrEmpty(contentTypeDefinition.GetSettings().Stereotype);
+    }
+
+    public static string GetStereotype(this ContentTypeDefinition contentTypeDefinition)
+    {
+        return contentTypeDefinition.GetSettings().Stereotype;
```

Comment:
Need to check if it has stereotype

Reply:
I don't this this is needed. This extension simply return `Stereotype` or `null`. If you need to check if `Stereotype` exists, you can use `HasStereotype`

Reply:
If it returns stereotype or `null`, `GetStereotypeOrDefault()` could make sense. It's similar to `Single()`, `SingleOrDefault` the first throws while the last one returns `null`

Reply:
Okay. I renamed it.